pkg/components/networkpolicy.go

```diff
@@ -0,0 +1,119 @@
+package components
+
+import (
+	corev1 "k8s.io/api/core/v1"
+	netv1 "k8s.io/api/networking/v1"
+	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
+	"k8s.io/apimachinery/pkg/util/intstr"
+	"k8s.io/utils/ptr"
+)
+
+func GetNetworkPolices(
```

Comment:
lets amend please templates/cluster-network-addons/VERSION/operator.yaml.in
with this manifest
no need to go gen it

Reply:
Done

Reply:
@oshoval why do you want to use template.in instead of the components.go? Mostly curious.

Reply:
Manifests are easier to maintain and read,
Maybe it should exists on other manifest yaml file if thats what you mean, the interesting thing here is that it should be in a manifest imo

fwiw i didnt finish to review yet
need to check for example if we need to make it dual (k8s / OpenShift), and then to think deeper, unless it is already dual